src/index_runner/indexers/indexer_utils.py

```diff
@@ -0,0 +1,28 @@
+from kbase_workspace_client import WorkspaceClient
+from kbase_workspace_client.exceptions import WorkspaceResponseError
+
+from ..utils.config import get_config
+
+
+def get_shared_users(data):
+    """
+    get shared users
```

Comment:
make the docstring something like "Returns a list of users that have read or write access to the given workspace object."